just_audio_background/lib/just_audio_background.dart

```diff
@@ -165,7 +171,7 @@ class _JustAudioPlayer extends AudioPlayerPlatform {
   late final _PlayerAudioHandler _playerAudioHandler;
 
   _JustAudioPlayer({required String id}) : super(id) {
-    _playerAudioHandler = _PlayerAudioHandler(id);
+    _playerAudioHandler = _PlayerAudioHandler._(id);
```

Comment:
This change is unnecessary since the class itself is private.

Reply:
Ah, yes, that remained from my first attempt, will re vert this.